### ISSUE TITLE
gaby feedback: overviews

### ISSUE THREAD

**Issue by tatianab** [golang/oscar#61]
Use this issue to provide feedback on the Gaby overviews page.

**Comment by hyangah** [comment]
/overview?q=https%3A%2F%2Fgithub.com%2Fgolang%2Fgo%2Fissues%2F70529&t=related_overview

`gabyhelp` posted "Related Code Changes" in its comment for the issue, but this related_overview processes only github issues and does not include gerrit cl docs.

**Comment by tatianab** [comment]
> /overview?q=https%3A%2F%2Fgithub.com%2Fgolang%2Fgo%2Fissues%2F70529&t=related_overview
> 
> `gabyhelp` posted "Related Code Changes" in its comment for the issue, but this related_overview processes only github issues and does not include gerrit cl docs.

@hyangah I'm seeing both CLs and issues on that page (they are not ordered by type though)

**Comment by hyangah** [comment]
@tatianab Indeed. I think my confusion came from 1) not grouped by type, 2) the related_overview page has total 5 docs while gabyhelp posted 10 links (6 are gerrit cls)
